CONTRIBUTING.md

```diff
@@ -381,7 +381,13 @@ file. To list all available commands, type `npm run`.
 [electron/electron-i18n]: https://github.com/electron/electron-i18n
 
 ## Need Help?
-
+### [Join](https://discord.gg/H6uTh7m) The Discord Server!
+Join the Discord server to get connected to thousands of Devs creating awesome projects with Electron.js everyday!
+  - Get Help! regarding any issue you are facing.
+  - Make new friends, working in the same space!
+  - Find some amazing projects to contribute to! or Help others with their issues!
+  - Still Reading for reasons to Join? duh! Just [Join](https://discord.gg/H6uTh7m)  Already!
```

Comment:
```suggestion
Join the Discord server to get connected to thousands of devs creating awesome projects with Electron
  - Get help regarding any issue you are facing.
  - Make new friends, working in the same space
  - Find amazing projects to contribute to, or help others with their issues
```

Reply:
Yes, sure, as it is much more compatible, with the rest of the file, it's great. 
Thanks for the changes.